aisynphys/connectivity.py

```diff
@@ -521,4 +535,54 @@ def err_fn(cls, params, *args):
         model = cls(pmax, size)
         return -model.likelihood(x, conn)
 
+def recip_connectivity_profile(cp_results_1, cp_results_2, bin_edges):
+    """
+    Given connection probability results of two pair groups what is the normalized probability of finding a reciprocal cnx 
+    
+    Parameters
+    ----------
+    cp_results_1 : dict
+        output of measure_connectivity for first connection type (A->B)
+    cp_results_2 : dict
+        output of measure connectivity for second connection type (B->A)
```

Comment:
I don't really get why you want to run measure_connectivity on a-b, b-a only to access the lists of pairs and distances? I think it would make sense just to pass in the list of pairs instead.

Reply:
I suppose I was trying to make it work with our other functions but could do what you suggest.

Reply:
So I guess the other reason was because I use other keys from the results dict to plug into `connectivity_profile` so that we're not constantly re-creating those arrays for various other analyses